lib/lita/handlers/jira_issues.rb

```diff
@@ -80,6 +82,26 @@ def priority(data)
       def issue_link(key)
         "\n#{config.url}/browse/#{key}"
       end
+
+      def silenced?(key)
+
+        if config.issue_ttl == 0
+          log.debug("JiraIssues: issue_ttl is set to 0, will post every matched issue to chat")
+          return false
+        end
+
+        current_ttl = redis.ttl(key).to_i
+
+        if current_ttl > 0
+          log.debug("JiraIssues: Key expiration not met for #{key}, will not reprompt for #{current_ttl} seconds")
+          true
+        else
+          redis.setex(key, config.issue_ttl, key)
```

Comment:
Personally I find it sometimes annoying when ruby is "to concise", and as such I tend to avoid single line conditionals whenever possible.  I am of the opinion that they can often confuse non-rubyist's, but of course that is just my opinion.  

This is your project and I am the guest here.  If you want me to refactor it I am happy to do so :)